daemon/cmd/daemon_main.go

```diff
@@ -971,6 +971,10 @@ func init() {
 	flags.Bool(option.EnableCustomCallsName, false, "Enable tail call hooks for custom eBPF programs")
 	option.BindEnv(option.EnableCustomCallsName)
 
+	flags.Bool(option.NetfilterCompatibleMode, false, "If set to true, guarantees that traffic will pass through netfilter and that iptable rules will be enforced. This mode may reduce network throughput. If set to false (default), there are no guarantees: some traffic may skip netfilter.")
+	flags.MarkHidden(option.NetfilterCompatibleMode)
```

Comment:
maybe add a TODO here describing conditions when we can un-hide this flag?

Reply:
I have updated this in GH [issue](https://github.com/cilium/cilium/issues/16166)

Reply:
Update GH [issue](https://github.com/cilium/cilium/issues/16166), to unhide this flag. Once we have all the fixes in.